connect_four.py

```diff
@@ -1,14 +1,18 @@
 
-class Board:
+class ConnectFourBoard:
+    class ColorEnum(Enum):
+        Y = 1
+        R = 2
    
     def __init__(self, c: int = 7, r: int = 6, w: int = 4) -> None:  
```

Comment:
This is kind of subjective, but I think it's better to avoid abbreviations. At my past team at facebook I would get a comment every time i had an abbreviation.